node/api/server.go

```diff
@@ -64,27 +65,13 @@ func PodHandler(p PodHandlerConfig, debug bool) http.Handler {
 			WithExecStreamIdleTimeout(p.StreamIdleTimeout),
 		),
 	).Methods("POST", "GET")
-	r.NotFoundHandler = http.HandlerFunc(NotFound)
-	return r
-}
 
-// PodStatsSummaryHandler creates an http handler for serving pod metrics.
-//
-// If the passed in handler func is nil this will create handlers which only
-//  serves http.StatusNotImplemented
-func PodStatsSummaryHandler(f PodStatsSummaryHandlerFunc) http.Handler {
```

Comment:
Let's not delete existing functions.
We can keep the old one for attaching to other